### ISSUE TITLE
Pagination with hits_per_page to limit per page but got error

### ISSUE THREAD

**Issue by mech** [meilisearch/meilisearch-rails#183]
I am trying to limit the pagination using hits_per_page with Kaminari:

```ruby
Candidate.search("", page: 1, hits_per_page: 5, sort: ["name:asc"])
```

But was given this error:

```
MeiliSearch::ApiError: 400 Bad Request - Json deserialize error: unknown field `hitsPerPage`, expected one of `q`, `offset`, `limit`, `attributesToRetrieve`, `attributesToCrop`, `cropLength`, `attributesToHighlight`, `showMatchesPosition`, `filter`, `sort`, `facets`, `highlightPreTag`, `highlightPostTag`, `cropMarker` at line 1 column 21. See https://docs.meilisearch.com/errors#bad_request.
```

Removing the `hits_per_page` and I got back the first 20 records of the first page.

I even replace it with limit, but no luck:

```ruby
# Replacing to limit don't work also
Candidate.search("", page: 1, limit: 5, sort: ["name:asc"])
```

Is there any special options we are not aware of? Or is pagination not working and need to wait for v0.29

- OS: macOS 12.5
- Meilisearch server version: v0.28.1
- meilisearch-rails version: v0.7.1
- Rails version: v7.0.3.1

**Comment by brunoocasali** [comment]
Hi @mech I found what happened, I'll provide a fix soon!

**Comment by brunoocasali** [comment]
@mech I created a branch with the fix, I need to have my PR approved to release a new version.

So while I wait for the approval, you have some options:

- You can reference this branch directly in your Gemfile
- You can change from `hits_per_page` to `hitsPerPage`

Thanks for using Meilisearch ❤️

**Comment by mech** [comment]
Wow tks, hitsPerPage does work. I will use this temporarily.

**Comment by brunoocasali** [comment]
It will be released soon as 0.7.2!

**Comment by mech** [comment]
I am puzzle, when I do a search, it appear it is querying all **200** records no matter which page I am on.

Doing `Candidate.ms_search("", page: 2, hitsPerPage: 3)` will result in such SQL:

```sql
SELECT * FROM "candidates" WHERE id IN ($1, $2, ...... $200)
```

Is this normal?

**Comment by brunoocasali** [comment]
Yes, it is normal because Meilisearch does not provide a way to know the total hits yet. 
@curquiza explained this here: https://github.com/meilisearch/meilisearch-rails/issues/138#issuecomment-1144747998

It is something we plan to improve in the future, in fact, we already have a prototype for this https://github.com/meilisearch/meilisearch/discussions/2635

**Comment by mech** [comment]
Tks for clarification, will play with the new prototype 👍
